pages/api/waitlist/index.ts

```diff
@@ -19,12 +20,22 @@ export default async function handle(
     req: NextApiRequest,
     res: NextApiResponse,
 ): Promise<void> {
+    const session = await getSession({ req });
+
+    const parentId = session.id as number;
+    if (!parentId) {
+        return ResponseUtil.returnBadRequest(
+            res,
+            "No user id stored in session",
+        );
+    }
+
     switch (req.method) {
         // Retrieving a waitlist by passing in the classId and parentId.
         case "GET": {
             const input = req.body as WaitlistInput;
 
-            if (input.classId && input.parentId) {
+            if (input.classId && parentId) {
```

Comment:
so I did this because when I logged `input`, I got undefined back -- the way I got `parentId` is in parallel with how it's gotten in `api/enroll/child`; not sure if the code here for `api/waitlist` is different for a reason ? let me know if i should explain better lol

Reply:
also not sure why `input` is undefined -- I can keep trying to debug this if we want, or I can fix it so that the parentId and classId are retrieved the same way it is in `api/enroll/child`

Reply:
wait what does the console think `req.body` is? I feel like it might be a bad request.

Reply:
i think it's undefined, i can check again later